assets/src/edit-story/components/form/toggleButton.js

```diff
@@ -110,12 +110,8 @@ function ToggleButton({
   ...rest
 }) {
   const toggle = () => onChange(!value);
-
-  // We unfortunately have to manually assign this listener, as it would be default behaviour
-  // if it wasn't for our listener further up the stack interpreting enter as "enter edit mode"
-  // for text elements. For non-text element selection, this does nothing, that default beviour
-  // wouldn't do.
   const inputRef = useRef();
+  // <enter> doesn't normally toggle checkboxes, but we'd like it to
```

Comment:
@miina, TIL that checkboxes aren't normally toggled by pressing <kbd>enter</kbd>. But we'd like it to with our buttons (as they're more like _buttons_ than _inputs_), so re-adding the handler here.